blocks/header/header.css

```diff
@@ -346,4 +329,21 @@ header nav[aria-expanded='true'] .nav-tools li {
     box-shadow: 0 0 5px var(text-color);
     z-index: 5;
   }
-}
\ No newline at end of file
+}
+
+header nav::after {
+  content: "★ Honor being honorable ★";
```

Comment:
Shouldn't this catchphrase be part of the indexable content so in the HTML document ?

Reply:
Possible, but that hasn't changed so I guess you could create a new issue for that one.

Reply:
Well I'm not sure if it's an issue in the first place. Maybe it's not worth fixing.